core/new-gui/src/app/workspace/service/validation/validation-workflow.service.ts

```diff
@@ -158,8 +158,13 @@ export class ValidationWorkflowService {
         .getLinkDeleteStream()
         .pipe(map(link => link.deletedLink))
     ).subscribe(link => {
-      this.updateValidationState(link.source.operatorID, this.validateOperator(link.source.operatorID));
-      this.updateValidationState(link.target.operatorID, this.validateOperator(link.target.operatorID));
+      if (
```

Comment:
can we do two if statements? so if only one operator is gone for some reason, the other's validation is still updated